Automattic/wp-calypso
# Automattic Components: Temporarily shim old calypso form label component

Related to https://github.com/Automattic/wp-calypso/pull/79256
Tracking issue https://github.com/Automattic/martech/issues/2023

## Proposed Changes
- At dale's suggestion, point the old form label component to the new one in @automattic/components, allowing us to transition from the old code to the new in smaller pieces and without risking updates to the old component in the interim.
- Remove old files

## Testing Instructions



* Check out this branch or use calypso.live
* VideoPress Setup
  * Navigate to calypso.localhost:3000/setup/videopress/options
  * <img width="1495" alt="Screenshot 2023-08-09 at 2 44 14 PM" src="https://github.com/Automattic/wp-calypso/assets/5414230/622e38b7-4b50-4be3-ba76-d94691502ca8">
* /me < Privacy
  * Navigate to calypso.localhost:3000/me/security/two-step
  * Click on "Register key"
  * <img width="1492" alt="Screenshot 2023-08-09 at 2 53 18 PM" src="https://github.com/Automattic/wp-calypso/assets/5414230/06793151-c507-418c-8eb9-652da5b14d2b">
  * Click on "Add new application password"
  * <img width="1493" alt="Screenshot 2023-08-09 at 2 50 24 PM" src="https://github.com/Automattic/wp-calypso/assets/5414230/181e2119-a008-4d9c-b3a9-d6d2dd218121">
* Domain Management
  * Navigate to calypso.localhost:3000/domains/manage/all/{SITE_SLUG_FOR_CUSTOM_DOMAIN}/add-dns-record/{SITE_SLUG_FOR_CUSTOM_DOMAIN}
  * <img width="1491" alt="Screenshot 2023-08-09 at 2 59 20 PM" src="https://github.com/Automattic/wp-calypso/assets/5414230/6dac4111-f991-4307-8529-d8ac3a012783">
* Importer
  * Navigate to calypso.localhost:3000/migrate/{SITE_SLUG}
  * Click on import from WordPress
  * <img width="1496" alt="Screenshot 2023-08-09 at 3 03 58 PM" src="https://github.com/Automattic/wp-calypso/assets/5414230/f00c2261-52b4-41f7-9b3c-4fa3d9008320">
* Hosting Configuration
  * Navigate to a site with a business plan calypso.localhost:3000/hosting-config/{SITE_SLUG}
  * <img width="1491" alt="Screenshot 2023-08-09 at 3 06 06 PM" src="https://github.com/Automattic/wp-calypso/assets/5414230/cc2e7d7b-caf4-4a4f-9bc1-562dac55ec24">
* Smoke test other instances of form label to ensure that they are rendered properly

## Pre-merge Checklist
